provisioning/service/src/Enrollments/X509Attestation.cs

```diff
@@ -247,7 +247,8 @@ public static X509Attestation CreateFromRootCertificates(string primary, string
         /// </remarks>
         /// <param name="primary">the <c>string</c> with the primary certificate. It cannot be <c>null</c> or empty.</param>
         /// <returns>The new instance of the X509Attestation.</returns>
-        /// <exception cref="ArgumentException">if the provide primary certificate is invalid.</exception>
+        /// <exception cref="ArgumentException">if the provided primary certificate is invalid.</exception>
+        /// <exception cref="CryptographicException">if the provided certificate is invalid.</exception>
         public static X509Attestation CreateFromCAReferences(string primary)
         {
             if (string.IsNullOrWhiteSpace(primary))
```

Comment:
This is the first case we can actually change to Argument.AssertNotNullOrWhiteSpace.